numpy/random/mtrand/mtrand.pyx

```diff
@@ -2645,7 +2645,7 @@ cdef class RandomState:
 
         fa = PyFloat_AsDouble(a)
         if not PyErr_Occurred():
-            if fa <= 0:
+            if fa < 0:
                 raise ValueError("a <= 0")
             return cont1_array_sc(self.internal_state, rk_pareto, size, fa,
```

Comment:
Similarly here, [`rk_pareto`](https://github.com/numpy/numpy/blob/master/numpy/random/mtrand/distributions.c#L624) will be doing a division by zero.